backend/slackbot/worker.js

```diff
@@ -94,11 +94,17 @@ async function getSSMParam(name) {
  */
 async function handleCommand(slackFields){
   const responseURL = slackFields['response_url'];
-  const parameters = await extractSlackParameters(slackFields);
+
+  let parameters;
+  try {
+    parameters = await extractSlackParameters(slackFields);
+  } catch (error) {
+    console.log(`Error Extracting Slack Parameters: ${error}`);
```

Comment:
Above on line 47, you have a try/catch around a call to `handleScheduledReport` - because you're handling the error here, that error will never make it up to the calling function. You have two options: remove the try/catch on line 47, or `throw` the error in the catch block here.